docs/mesos/vagrant-centos/units/docker.service.d/docker.conf

```diff
@@ -1,4 +1,4 @@
 [Service]
 EnvironmentFile=/etc/sysconfig/calico
 ExecStart=
-ExecStart=/usr/bin/docker daemon -H fd:// --cluster-store=etcd://${ETCD_AUTHORITY}
\ No newline at end of file
+ExecStart=/usr/bin/dockerd --cluster-store=etcd://${ETCD_AUTHORITY}
```

Comment:
Yeah I couldn't find details on what that did, but i remember seeing it in some Docker or CoreOS documentation somewhere, and I remember that not including it stops it from working -- so it was needed.